packages/react/src/Button/Button.module.css

```diff
@@ -118,11 +117,6 @@
   color: var(--brand-button-secondary-fgColor-rest);
 }
 
-.Button--secondary .Button__icon-visual,
-.Button--secondary .Button-arrow {
-  color: var(--brand-button-secondary-fgColor-rest);
-}
-
```

Comment:
@stamat do we need to remove these styles? Doesn't seem related to HCM (might be wrong?) It's caused a visual regression whereby the arrow is now blue:

<img width="285" alt="Screenshot 2024-12-04 at 12 43 50" src="https://github.com/user-attachments/assets/a8df3cc2-45ea-4cef-a963-c393a941b9ac">